readthedocs/doc_builder/python_environments.py

```diff
@@ -63,13 +68,27 @@ def delete_existing_venv_dir(self):
             ))
             shutil.rmtree(venv_dir)
 
-    def install_package(self):
-        if (self.config.python.install_with_pip or
-                getattr(settings, 'USE_PIP_INSTALL', False)):
+    def install_requirements(self):
+        for install in self.config.python.install:
+            self._run_install_step(install)
+
+    def _run_install_step(self, install):
```

Comment:
This method could be merged with the previous one. I don't see too much value on splitting here.

Reply:
I just don't like so many levels of indentations, I think there were more at the beginning